agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/telemetry/TelemetryClient.java

```diff
@@ -266,7 +292,7 @@ public BatchSpanProcessor getStatsbeatChannelBatcher() {
             LocalFileSender.start(localFileLoader, channel);
           }
 
-          statsbeatChannelBatcher = BatchSpanProcessor.builder(channel).build();
+          statsbeatChannelBatcher = BatchSpanProcessor.builder(channel).build("statsbeat");
```

Comment:
Statsbeat is MetricData too and it won't hit the queue capacity since it's using a much lower frequency.. every 15 mins and 24 hours.. but we do need a separate channel for statsbeat.. i wonder if we can keep statsbeat channel the old capacity since it doesn't apply to statsbeat metrics.

Reply:
nothing changed for the statsbeat channel